dotcom-rendering/src/web/components/FixedMediumFastXI.stories.tsx

```diff
@@ -0,0 +1,144 @@
+import { breakpoints } from '@guardian/source-foundations';
+import { trails } from '../../../fixtures/manual/trails';
+import { FixedMediumFastXI } from './FixedMediumFastXI';
+import { Section } from './Section';
+
+export default {
+	component: FixedMediumFastXI,
+	title: 'Components/FixedMediumFastXI',
+	parameters: {
+		chromatic: {
+			viewports: [
```

Comment:
Is this many breakpoints intentional?

Reply:
No, not intentional at all! This slipped in from a me copying old code, thanks for spotting